lib/CFRPageActions.jsm

```diff
@@ -49,7 +49,20 @@ class PageAction {
     this.container.onclick = this._handleClick;
   }
 
-  async show(notificationText, shouldExpand = false) {
+  _sendTelemetry(ping) {
+    // Note `useClientID` is set to true to tell TelemetryFeed to use client_id
+    // instead of `impression_id`. TelemetryFeed is also responsible for deciding
+    // whether to use `message_id` or `bucket_id` based on the release channel and
+    // shield study setup.
+    this._dispatchToASRouter({
+      data: {
+        type: "TELEMETRY",
+        data: Object.assign({useClientID: true, action: "cfr_user_event", source: "CFR"}, ping)
```

Comment:
Can you use the spread operator style i.e. `{useClientID: true, action: "cfr_user_event", source: "CFR", ...ping}`